Library/Homebrew/rubocops/bottle_block_cop.rb

```diff
@@ -1,20 +1,31 @@
+require_relative "./extend/formula_cop"
+
 module RuboCop
   module Cop
     module Homebrew
-      class CorrectBottleBlock < Cop
-        MSG = "Use rebuild instead of revision in bottle block".freeze
+      # This cop audits `bottle` block in Formulae
+      #
+      # - `rebuild` should be used instead of `revision` in `bottle` block
 
-        def on_block(node)
-          return if block_length(node).zero?
-          method, _args, body = *node
-          _keyword, method_name = *method
+      class CorrectBottleBlock < FormulaCop
+        MSG = "Use rebuild instead of revision in bottle block".freeze
 
-          return unless method_name == :bottle
-          check_revision?(body)
+        def on_formula(_node, _class_node, _parent_class_node, body)
+          check(body)
         end
 
         private
 
+        def check(body)
```

Comment:
Not totally clear to me what `body` is here; could you explain so we can maybe pick a better name?